packages/core/src/app/shipping/StaticConsignmentItemList.tsx

```diff
@@ -28,7 +38,10 @@ const StaticConsignmentItemList: FunctionComponent<StaticConsignmentItemListProp
 
             <ul>
                 {lineItems.map((item) => (
-                    <li key={item.id}>{`${item.quantity} x ${item.name}`}</li>
+                    <li key={item.id}>
+                        <strong>{item.quantity} x </strong>{item.name} 
+                        {renderProductOptionDetails(item)}
+                    </li>
```

Comment:
The content seems similar to the one in ConsignmentLineItem, we can move it to a separate component and use it a both places.